docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.0
 info:
-  version: "0.1"
+  version: "1.0.0"
```

Comment:
How should this version relate to the AP's version? Should it match? Or should it be independent?

Reply:
I would prefer independent. But all APIs should have the same version string.

However, I don't see the way to share the version string in swagger.